Ivanti/pulse-connect/ingest/parser.yml

```diff
@@ -0,0 +1,36 @@
+name: Pulse Connect Secure
+pipeline:
+  - name: kv_event
+    external:
+      name: kv.parse-kv
+      properties:
+        item_sep: '\s'
+  - name: field_extraction
+  - name: msg_parsing
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{kv_event.message.msg}}'
+        output_field: details
+        pattern: '(AUT31556: Unauthenticated request url %{URIPATHPARAM:url} %{GREEDYDATA}|%{GREEDYDATA})'
+      filter: "{{field_extraction.action.name == 'AUT31556'}}"
+  - name: url_extraction
+stages:
+  field_extraction:
+    actions:
+      - set:
+          action.name: "{{kv_event.message.msg.split(':')[0]}}"
+          log.priority: "{{kv_event.message.pri}}"
```

Comment:
Not an official ECS field. We could use `log.syslog.priority` instead, but it might not be a good idea since we aren’t facing a syslog event. IMHO we could drop this information.